src/ralph/scan/plugins/ssh_cisco_asa.py

```diff
@@ -104,12 +104,24 @@ def scan_address(ip_address, **kwargs):
     ssh = _connect_ssh(ip_address, SSH_USER, SSH_PASS)
     try:
         lines = ssh.asa_command(
-            "show version | grep (^Hardware|Boot microcode|^Serial|address is)"
+            "show version | grep (^Hardware|Boot microcode|address is)"
         )
+        lines2 = ssh.asa_command("show inventory")
     finally:
         ssh.close()
+
+    inventory = [item for item in lines2 if item != u'\r']
+    components = []
+    for parts in [i + j for i, j in zip(inventory[::2], inventory[1::2])]:
+        element = {}
+        for item in re.findall(r'(?:[^,,"]|"(?:\\.|[^"])*")+',
+                               re.sub('\r', ',', parts)):
+            k, v = item.split(":")
+            element[k.strip().lower().replace('"',
```

Comment:
Maybe it's me ;) but this:
`k.strip().lower().replace('"','')`
also could be assigned to descriptive variable.